### ISSUE TITLE
Please add druid forms

### ISSUE THREAD

**Issue by beren12** [exochron/MountJournalEnhanced#63]
gogo mount used to be able to use travel form when in combat, cat form indoors, and flying form. Could you add any of that, at least flight form support please?

**Comment by Knochenmarc** [comment]
I'm not quite sure what you exactly want.
Do you wish for the druid forms to be listed in the journal?
Or do you like to use them somehow with the summon random mount button?

Either way technically the forms are not in the mount system of blizzard. So therefore it's also not in the journal (yet).

**Comment by exochron** [comment]
I'm not quite sure what you exactly want.
Do you wish for the druid forms to be listed in the journal?
Or do you like to use them somehow with the summon random mount button?

Either way technically the forms are not in the mount system of blizzard. So therefore it's also not in the journal (yet).

**Comment by beren12** [comment]
Both. I prefer to almost always use the flight form, but cat and travel forms are better than running with feet and can be used indoors/in combat. Basically for druids would be great to have a button for "go the fastest way I can right now"

**Comment by Deviousdb1** [comment]
Yeah, this is not something the majority of people would be happy with. Would recommend just binding your random mount and your travel form to different keys, or use a modifier with a macro.
